packages/react/src/themes/colorUtils.ts

```diff
@@ -16,3 +21,23 @@ export const extendColorScheme = (
   })
   return result
 }
+
+export function pickValuesFromColorScheme<T extends Partial<ComponentAreaName>>(
```

Comment:
the intent of this method is to take specified areas from _each_ color scheme - in that case I would expect that even for the `default` one we would have only `foreground` and `background` for `Label`. However, types suggests that for `default` we still have all of the areas originally defined:

![image](https://user-images.githubusercontent.com/9024564/57776625-ab6dae80-7728-11e9-9f68-03e477149065.png)

It is necessary that we would marry types and actual implementation, in a way that real code will guarantee type correctness

Reply:
This was fixed: 
![image](https://user-images.githubusercontent.com/4512430/57780532-b6c0da00-7728-11e9-9e3c-dcb2f1478641.png)

I had to introduce `StrictColorSchemeMapping` as a new type, plus I had prev generic that I was not using... It's behaving as expected now. Thanks for the catch!